packaging/tests/tas-restapi/src/test/java/org/alfresco/rest/rules/CreateRulesTests.java

```diff
@@ -252,6 +253,20 @@ public void createRulesWithOneError()
         restClient.assertLastError().containsSummary("Rule name is a mandatory parameter");
     }
 
+    /** Check we can create a rule. */
+    @Test (groups = { TestGroup.REST_API, TestGroup.RULES, TestGroup.SANITY })
+    public void createRuleAndIncludeFieldsInResponse()
+    {
+        RestRuleModel ruleModel = createRuleModel("ruleName");
+
+        RestRuleModel rule = restClient.authenticateUser(user).withCoreAPI().usingNode(ruleFolder).usingDefaultRuleSet()
+                                       .include("isShared")
+                                       .createSingleRule(ruleModel);
+
+        restClient.assertStatusCodeIs(CREATED);
+        rule.assertThat().field("isShared").isNotNull();
```

Comment:
Is `isShared` a mandatory/always returned in the response field? If yes, I think we can hint that is tested in the comment/description and maybe update one of the GET tests to include `isShared` assert

Reply:
`isShared` is an optional field that needs requesting with `?include=isShared`

Reply:
Thanks. In that case I think we should update the description/comments of the existing tests you've updated with the` isShared` functionality to make it clear that it's been tested there as currently the comments only state that only basic functionality is tested.

Reply:
Good suggestion - done in 64b3e61e1cdf7fae338d8158ec106278fd242bf5